lib/core/app_review.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:in_app_review/in_app_review.dart';
+
+class AppReviewHelper {
+  static void showAppReview() async {
+    await InAppReview.instance.isAvailable();
```

Comment:
you must check if isAvailable return true first then call requestReview 
and I think this method could return Future<bool> true when isAvailable is true and request review is called, 
otherwise return false

Reply:
done